### PR TITLE
adds dockerfile and ci/cd

### PR DESCRIPTION
**Pull request by ajansari95**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          go-version: 1.19
+
+      - uses: actions/checkout@v3
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.48.0
+          args: --timeout=3m
```
Comment (ajansari95) [inline .github/workflows/lint.yaml:25]:
```suggestion
          args: --timeout=3m
          
```

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push evinced
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          tags: /evince:latest
```
Comment (joe-bowman) [inline .github/workflows/docker.yaml:28]: will this automatically be prefixed by quicksilverzone given that's the repo it is pushed to?

Comment (ajansari95) [reply .github/workflows/docker.yaml:28]: yup, we need the secrets it would be done.

**Review by zanicar — APPROVED** [review]
lgtm

**Pull request merged**